addons/website_forum/static/src/js/website_forum.js

```diff
@@ -342,11 +340,12 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                     }
                     $(elem).nextAll('.flag_validator').removeClass('d-none');
                 } else if (data.success === 'post_flagged_non_moderator') {
-                    const forumAnswer = elem.closest('.forum_answer');
-                    elem.innerText = _t(' Flagged');
-                    elem.prepend(child);
-                    forumAnswer.fadeIn(1000);
-                    forumAnswer.slideUp(1000);
+                    this.displayNotification({
+                        message: _t("Moderator has been notified about this post."),
+                        title: _t("Post has been flagged"),
```

Comment:
Moderators will be notified about this post.

"The post has been successfully flagged" `or` "Post successfully flagged" `or` "The post was just flagged" `or` "Post flagged" look better than the current proposition, if I may raise this point.